mesonbuild/wrap/wraptool.py

```diff
@@ -212,7 +212,7 @@ def status(options: 'argparse.Namespace') -> None:
         try:
             (current_branch, current_revision, _, _, _) = get_current_version(w)
         except Exception:
-            print('Wrap file not from wrapdb.', file=sys.stderr)
+            print('Wrap file "{}" not from wrapdb.'.format(name), file=sys.stderr)
```

Comment:
```suggestion
            print(f'Wrap file {name!r} not from wrapdb.', file=sys.stderr)
```

f-strings are faster and often more concise than format functions.